db/migrate/20171122192715_add_address_to_users.rb

```diff
@@ -0,0 +1,13 @@
+class AddAddressToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :cpf, :string
```

Comment:
If these columns are going to be required, you should add `null: false` to each one so they are required in the database as well.